packages/richtext-lexical/src/index.ts

```diff
@@ -39,6 +39,7 @@ let checkedDependencies = false
 
 export function lexicalEditor(props?: LexicalEditorProps): LexicalRichTextAdapterProvider {
   return async ({ config, isRoot, parentIsLocalized }) => {
+    // is this dependency check still necessary?
```

Comment:
I think so - people still have the ability to install lexical packages in their project manually. If that happens, they will experience errors in the admin panel due to mismatching lexical versions (e.g. "cannot find active editor")